skfda/preprocessing/dim_reduction/feature_extraction/_fpca.py

```diff
@@ -375,7 +378,7 @@ def _fit_grid(
         ).T
```

Comment:
Line 373: basis_matrix += regularization_matrix

Reply:
That would change the basis.